apps/daimo-mobile/src/action/key.ts

```diff
@@ -9,13 +9,13 @@ import { createEnclaveKey, loadEnclaveKey } from "../logic/enclave";
 function getKeySecurityMessage(hwSecLevel: ExpoEnclave.HardwareSecurityLevel) {
   switch (hwSecLevel) {
     case "SOFTWARE":
-      return "Key generated in software";
+      return "Device key generated in software";
     case "TRUSTED_ENVIRONMENT":
-      return "Key generated in trusted hardware";
+      return "Device key generated in trusted hardware";
     case "HARDWARE_ENCLAVE":
       return Platform.OS === "ios"
-        ? "🔒  Key generated in Secure Enclave"
-        : "🔒  Key generated in hardware enclave";
+        ? "🔒  Device key generated in Secure Enclave"
+        : "🔒  Device key generated in hardware enclave";
```

Comment:
idk if "Device key" is less confusing than "Key"

Reply:
i think we have an opportunity for a nice "Vault" ui in future. something like

> ✅ Device key · secure enclave
> ✅ Passkey backup

...then can show a partial version of the "vault" with just that first checkmark to during onboarding

user gets second checkmark later, once they add a passkey

goal of the UI is to visually communicate how our self-custody works, and assure the user their account is safe

Reply:
moved to master issue for onboarding redesign #606